### DISCUSSION TITLE
How to custom maxContentWidth?

### DISCUSSION THREAD

**Discussion by tanshiqi** [filamentphp/filament#7811] [Help]
Correspond to [Tailwind's max-width scale](https://tailwindcss.com/docs/max-width). There are more options. Especially the "max-w-screen-2xl"(1536px). It's wider than 7xl and looks more comfortable on wide screens.

How to add this in Filament 3 ?

**Comment by danharrin** [comment]
You should PR it

**Comment by tanshiqi** [comment]
#7834 Add more options to maximum content width.
